tests/func/test_data_cloud.py

```diff
@@ -203,7 +203,8 @@ def test_verify_hashes(tmp_dir, scm, dvc, mocker, tmp_path_factory, local_remote
     # Removing cache will invalidate existing state entries
     dvc.cache.local.clear()
 
-    dvc.config["remote"]["upstream"]["verify"] = True
```

Comment:
In-memory changes to config can't work between command sessions, the logic is really the same as with us resetting index, because other commands/instances could modify the config while we are not holding a lock. Same in changed tests below.